.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: node_js
+
 node_js:
   - "8.9.4"
-cache: yarn
+
 script:
-  - yarn build
-  - yarn test-js
+  - npm run build
+
```

Comment:
I think we dont have to test the compiled js files if the ts test script succeeds!